modules/auxiliary/scanner/http/git_scanner.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'HTTP Git Scanner',
+      'Description' => 'Detect git directories and files and analize its content.',
+      'Author'       => ['t0nyhj'],
+      'License'     => MSF_LICENSE
+    )
+
+    register_options(
+      [
+        OptString.new('PATH', [ true,  "The test path to .git directory", '/'])
+
+      ], self.class)
+
+    register_advanced_options(
+      [
+        OptInt.new('ErrorCode', [ true, "Error code for non existent directory", 404]),
+        OptPath.new('HTTP404Sigs',   [ false, "Path of 404 signatures to use",
+            File.join(Msf::Config.data_directory, "wmap", "wmap_404s.txt")
+          ]
+        )
+
+      ], self.class)
+  end
+
+  def run_host(target_host)
+    conn = true
+    ecode = nil
+    emesg = nil
+
+    tpath = normalize_uri(datastore['PATH'])
+
+    ecode = datastore['ErrorCode']
+    vhost = datastore['VHOST'] || wmap_target_host
+
+    #
+    # Detect error code
+    #
+    begin
+      randdir = Rex::Text.rand_text_alpha(5).chomp + '/'
+      res = send_request_cgi({
+        'uri'  		=>  tpath+randdir,
+        'method'   	=> 'GET',
+        'ctype'		=> 'text/html'
+      })
+
+      return if not res
+
+      tcode = res.code.to_i
+
+      # Look for a string we can signature on as well
+      if(tcode >= 200 and tcode <= 299)
+
+        File.open(datastore['HTTP404Sigs'], 'rb').each do |str|
+          if(res.body.index(str))
+            emesg = str
+            break
+          end
+        end
+
+        if(not emesg)
+          if datastore['VERBOSE']
+          vprint_status :level => :verror, :ip => ip, :msg => "['#{target_host}'] Using first 256 bytes of the response as 404 string '#{res.code}'"
```

Comment:
datastore['VERBOSE'] is already checked on `vprint_status`. You don't need to check it before calling `vprint_status`. On the other hand use parentheses around the `vprint_status` arguments (https://github.com/bbatsov/ruby-style-guide#no-dsl-parens).